src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -520,12 +515,13 @@ export default {
 		const cachedLoan = readLoanFragment({
 			apollo: this.apollo,
 			loanId: this.loanId,
-			fragment: loanCardFieldsFragment,
+			fragment: loanCardFieldsExtendedFragment,
 		});
 		if (cachedLoan) {
 			this.loan = cachedLoan;
 			this.isLoading = false;
 		}
+		this.loanCallouts = loanCallouts(this.loan, this.categoryPageName);
```

Comment:
This should probably be moved into a method and called in the two places it's needed in this component. Just in case the props of this method call changes, we can change in one place.